### DISCUSSION TITLE
Option to disable direct pushes to branch but still allow merging PRs overriding checks?

### DISCUSSION THREAD

**Discussion by midgleyc** [community/community#29062] [Pull Requests]
I have a branch, `main`, with a branch policy requiring particular checks to pass before a PR can be merged.

On a PR an administrator has the option:
> Merge without waiting for requirements to be met (bypass branch protections)

An administrator can also push directly to `main`. I'd like to disable this, because it's happened accidentally a few times. One way I can do this is to check "Include administrators" in the branch protection settings. However, this also disables the checkbox to bypass branch protections, which can be convenient in case the tests are timing out (e.g. taking over an hour on Ubuntu while Mac / Windows pass fine).

Can I disable direct branch push without losing the checkbox?

**Comment by wirew0rm** [comment]
Did you find a solution or some further discussions to this in the meantime? I have the same problem and at first thought I had miss-configured something. This is a really strange default setting IMO.
